enterprise/internal/campaigns/changeset_history.go

```diff
@@ -221,3 +230,32 @@ func reduceReviewStates(statesByAuthor map[string]campaigns.ChangesetReviewState
 	}
 	return selectReviewState(states)
 }
+
+// initialExternalState infers from the changeset state and the list of events in which
+// ChangesetExternalState the changeset must have been when it has been created.
+func initialExternalState(ch *campaigns.Changeset, ce ChangesetEvents) campaigns.ChangesetExternalState {
+	readyForReview := true
```

Comment:
Easier to understand, since it doesn't really have anything to do with "ready for review", right? (You can also review a draft PR)

```suggestion
	open := true
```

Reply:
Yeah, GitHub calls a non draft PR "ready for review". But since this is GitHub specific and this method ought to be general, I renamed it to `open`.